src/Card-Description.js

```diff
@@ -1,17 +1,65 @@
 import React, { Component } from 'react';
 import cardDescriptionStyle from './Card-Description.scss';
+import Rescue from './Rescue.js';
 
 export default class CardDescription extends Component {
+  constructor() {
+    super();
+    this.state = {
+      displayDescription: true,
+      currentRescue: []
+    }
+  }
+
+  toggleRescue = (event) => {
+    this.filterRescue();
+    this.setState({displayDescription: !this.state.displayDescription});
+  }
+
+  filterRescue = () => {
+    let matchedRescue = this.props.rescues.find((currentRescue) => {
+      return this.props.description.rescue.toLowerCase() === currentRescue.name.toLowerCase()
+    })
+    console.log('matchedRescue', matchedRescue);
+    this.state.currentRescue.push(matchedRescue);
```

Comment:
Simply pushing to your state isn't going to trigger a re-render. Remember if we want to modify anything in state, we must always use `this.setState`.